doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -12,13 +12,15 @@ The ``Scheme`` command is a high-level tool for generating automatically
 (possibly mutual) induction principles for given types and sorts. Its
 syntax follows the schema:
 
-.. cmd:: Scheme @ident := Induction for @ident Sort sort {* with @ident := Induction for @ident Sort sort}
+.. cmd:: Scheme @ident__1 := Induction for @ident__2 Sort sort {* with @ident__i := Induction for @ident__j Sort sort}
 
-where each `ident'ᵢ` is a different inductive type identifier 
-belonging to the same package of mutual inductive definitions. This
-command generates the `identᵢ`s to be mutually recursive
-definitions. Each term `identᵢ` proves a general principle of mutual
-induction for objects in type `identᵢ`.
+  This command is a high-level tool for generating automatically
+  (possibly mutual) induction principles for given types and sorts.
+  Each :n:`ident__j` is a different inductive type identifier belonging to
+  the same package of mutual inductive definitions.
+  The command generates the :n:`ident__i`\s` to be mutually recursive
```

Comment:
There's a spurious backtick *after* the `s`

Reply:
It actually came from you haha. ;)